.pre-commit-config.yaml

```diff
@@ -63,7 +55,8 @@ repos:
   hooks:
   - id: flake8
     additional_dependencies:
-    - flake8-cognitive-complexity ~= 0.1.0
+    - flake8-cognitive-complexity == 0.1.0
```

Comment:
Why did you change the version specifier in the unrelated entry? Revert it.

Reply:
Okay

Reply:
Marking this as resolved.